rules/concepts.md

```diff
@@ -48,6 +48,151 @@ In some cases, it is useful to have a rule that is only manually triggered.
+There are some more events, but it's not necessary to mention them here.
+
+In contrast to this, a trigger identifies one of those events and makes the rule run.
+:::
+
+Here are the details for each trigger type:
+
+### **Item**-based Triggers
+
+You can listen to commands for a specific item, on status updates or on status changes (an update might leave the status unchanged).
+You can decide whether you want to catch only a specific command/status or any.
+
+A simplistic explanation of the differences between `command` and `update` can be found in the article about [openHAB core actions](/docs/configuration/actions.html#event-bus-actions).
+
+**Item**-based triggers provide some information, e.g. the received command or the received state update, see [Available Values](#available-values).
+
+When using a `received command` trigger, the rule might trigger **before** the Item's state is updated.
+Therefore, if the rule needs to know what the command was, there is an [Available Value](#available-values).
+
+### **Group**-based Triggers
+
+As with Item based event-based triggers discussed above, you can listen for commands, status updates, or status changes on the direct (but not the indirect members of a nested subgroup) members of a given Group.
+You can also decide whether you want to catch only a specific command/status or any.
+
+The [Available Values](#available-values) are populated using the Item that caused the Group trigger to fire and that Item's name is stored in an additional value.
+
+Also, as with **Item**-based triggers, when using a `received command` trigger, the rule might trigger **before** the Item's state is updated.
+Therefore, if the rule needs to know what the command was, there is an [Available Value](#available-values).
+
+### **Time**-based Triggers
+
+Dependening on the rule language/engine, there are some predefined expressions you can use, but universally supported are only [cron expressions](https://www.quartz-scheduler.org/documentation/quartz-2.2.2/tutorials/tutorial-lesson-06.html).
+
+Please be aware that openHAB is using the [Quartz Scheduler](https://www.quartz-scheduler.org/documentation/quartz-2.2.2/), which is using a slighly different form than the Unix cron scheduler.
+
+**Time**-based triggers do not provide any information in the [Available Values](#available-values).
+
+A Quartz cron expression takes the form of six or optionally seven fields:
+
+1. Seconds
+2. Minutes
+3. Hours
+4. Day of Month
+5. Month
+6. Day of Week
+7. Year (optional field)
+
+You may use the generator at [FreeFormatter.com](https://www.freeformatter.com/cron-expression-generator-quartz.html) or the openHAB WebUI rule setup to generate your cron expressions.
+
+### **Channel**-based Triggers
+
+Some add-ons provide trigger channels.
+Compared with other types of channels, a trigger channel provides information about discrete events, but does not provide continuous state information.
```

Comment:
"Compared with other types of channels" doesn't add much to this sentence. I think it reads better without the lead in.

Reply:
👍